src/commands/admin/welcome.js

```diff
@@ -1,103 +1,104 @@
+        await interaction.reply({
+            content: "You don't have permission to do that!",
+            ephemeral: true,
+        });
+        return;
+    }
 
-		// If addon for welcome is missing create it
-		if (!data.addons.welcome) {
-			data.addons.welcome = {
-				enabled: true,
-				channel: '',
-				title: '',
-				message: '',
-				image: false,
-				embed: false,
-			};
-			data.markModified('addons.welcome');
-			await data.save();
-		}
+    const channel = interaction.options.getChannel('channel');
+    const title = interaction.options.getString('title');
+    const message = interaction.options.getString('body');
+    const image = interaction.options.getString('image');
+    const enable = interaction.options.getBoolean('enable');
 
-		if (typeof enable !== 'undefined' && enable !== null) {
-			data.addons.welcome.enabled = enable;
-			data.markModified('addons.welcome');
-			await data.save();
-		}
+    let data = await guildService.fetchGuild(interaction.guild.id);
 
-		if (typeof channel !== 'undefined' && channel !== null) {
-			data.addons.welcome.channel = channel.id;
-			data.markModified('addons.welcome');
-			await data.save();
-		}
+    // If addon for welcome is missing create it
+    if (!data.addons.welcome) {
+        data.addons.welcome = {
+            enabled: true,
+            channel: '',
+            title: '',
+            message: '',
+            image: false,
+            embed: false,
+        };
+        data.markModified('addons.welcome');
+        await data.save();
+    }
 
-		if (typeof image !== 'undefined' && image !== null) {
-			data.addons.welcome.image = image;
-			data.markModified('addons.welcome');
-			await data.save();
-		}
+    if (typeof enable !== 'undefined' && enable !== null) {
+        data.addons.welcome.enabled = enable;
+        data.markModified('addons.welcome');
+        await data.save();
+    }
 
-		if (typeof title !== 'undefined' && title !== null) {
-			data.addons.welcome.title = title;
-			data.markModified('addons.welcome');
-			await data.save();
-		}
+    if (typeof channel !== 'undefined' && channel !== null) {
+        data.addons.welcome.channel = channel.id;
+        data.markModified('addons.welcome');
+        await data.save();
+    }
 
-		if (typeof message !== 'undefined' && message !== null) {
-			data.addons.welcome.message = message;
-			data.markModified('addons.welcome');
-			await data.save();
-		}
+    if (typeof image !== 'undefined' && image !== null) {
+        data.addons.welcome.image = image;
+        data.markModified('addons.welcome');
+        await data.save();
+    }
 
-		await interaction.reply({ content: 'Settings saved!', ephemeral: true });
-	},
+    if (typeof title !== 'undefined' && title !== null) {
+        data.addons.welcome.title = title;
+        data.markModified('addons.welcome');
+        await data.save();
+    }
+
+    if (typeof message !== 'undefined' && message !== null) {
```

Comment:
Similar to above, does this line need to be an equality check on `undefined` and `null` types? Or could it be simplified to `!message`?